src/components/MasterItemsSelect.js

```diff
@@ -6,8 +6,10 @@ class MasterItemsSelect extends Component {
     super(props);
     this.handleChange = this.handleChange.bind(this);
     this.handleChangeCategory = this.handleChangeCategory.bind(this);
+    const filteredItems = this.filterItems(this.props.masterItems[0].category);
     this.state = {
-      filteredItems: this.filterItems(this.props.masterItems[0].category)
+      filteredItems,
+      selectedItemId: filteredItems[0].id
```

Comment:
Hmm yeah.. see other comment re: value of select. That's the only place this is being used right? Why do we need it now but didn't before?